### PR TITLE
Grammar + other changes [README.md] 

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26004)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Windows Package Manager Community Repository
-[![Gitter](https://img.shields.io/badge/chat-on%20gitter-yellow.svg)](https://gitter.im/Microsoft/winget-pkgs)
 
 This repository contains the manifest files for the **Windows Package Manager** default source.  You are highly encouraged to submit manifests for your favorite application.
->Note: At this time installers must either be MSIX, MSI or .exe application installers. Standalone or portable executables, compressed .zip files, and fonts are not currently supported.
+> Note: At this time installers must either be MSIX, MSI or .exe application installers. Standalone or portable executables, compressed .zip files, and fonts are not currently supported.
 
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage.
 
+[![Gitter](https://img.shields.io/gitter/room/Microsoft/winget-pkgs?style=for-the-badge)](https://gitter.im/Microsoft/winget-pkgs)
```
Comment (jedieaston) [inline README.md:8]: I like the badge that was there :( 

Is there a reason it needs to change?

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vedantmgoyal2009) [reply README.md:8]: I feel that it would be better there. The badge doesn't need to be above the description of the repository.

Comment (jedieaston) [reply README.md:8]: It seems very rare to me that people put badges not at the top of the README. I've never actually seen it except here, it's designed so you can see them without scrolling. That's why the images are usually less than 100x100.

**Comment by vedantmgoyal2009** [comment]
@jedieaston I have done the way you like it but now.

@denelon I hope you will agree with the changes I made to Pipeline Health badges.

**Comment by denelon** [comment]
I don't think we need the prefix on the health badges. They would become inconsistent with the comments in the PR.

**Comment by denelon** [comment]
It's fine. Nobody had any complaints since you didn't modify the CODE_OF_CONDUCT.md.

**Review by denelon — APPROVED** [review]

**Pull request merged**